aea/helpers/dialogue/base.py

```diff
@@ -241,6 +251,25 @@ def get_dialogue(self, msg: Message, agent_addr: Address) -> Dialogue:
         :return: the dialogue
         """
 
+    @staticmethod
+    def empty_dialogue(
+        opponent_address: Address = "", starter_address: Address = ""
+    ) -> Dialogue:
+        """
+        Create an empty dialogue, optionally with the supplied opponent address and dialogue starter address.
+
```

Comment:
What is the purpose of an empty dialogue?

Reply:
Used in the `is_permitted_for_new_dialogue` method